certs/certs.c

```diff
@@ -28,7 +28,10 @@ const char certificates[] =
 "ksLi4xaNmjICq44Y3ekQEe5+NauQrz4wlHrQMz2nZQ/1/I6eYs9HRCwBXbsdtTLS\r\n"
 "R9I4LtD+gdwyah617jzV/OeBHRnDJELqYzmp\r\n"
 "-----END CERTIFICATE-----\r\n"
-/*DigiCert Global Root CA*/
+#ifndef LEAN_CERTS
+//Note: for the ESP8266, other certs are defined out to save on space. 
```

Comment:
>the ESP8266 [](start = 12, length = 11)

this should likely be more generic like "devices with limited resources" as an indication and potential option to set for other boards/developers.

Reply:
Done.